src/Nethermind/Nethermind.Network/ProtocolsManager.cs

```diff
@@ -197,10 +194,6 @@ public void AddProtocol(string code, Func<ISession, IProtocolHandler> factory)
                 {
                     var ethHandler = version switch
                     {
-                        62 => new Eth62ProtocolHandler(session, _serializer, _stats, _syncServer, _txPool, _gossipPolicy, _logManager),
```

Comment:
Are we removing the files also?

Reply:
No, we need these files. `Eth66ProtocolHandler` is extending `Eth65ProtocolHandler`, which is extending  `Eth64ProtocolHandler` etc. We are not creating instances of old protocols, but files are still needed to use methods from them